hera_pspec/pstokes.py

```diff
@@ -269,3 +269,48 @@ def construct_pstokes(dset1, dset2, pstokes='pI', run_check=True, antenna_nums=N
         uvdS.check()
 
     return uvdS
+
+
+def choose_inp_dsets(dsets, pstokes):
```

Comment:
I think this function probably needs a `pol` in its name somewhere, as it's all about selecting based on polarisation. Also, can ambiguities arise, when more than one UVData provides the necessary polarizations?

Reply:
Okay! I'll put a `pol` in the name. Yes that ambiguity can arise and its resolution is stated in the docstring as `If a single UVData object 
    has multiple polarizations, this function only uses its first.`